naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftCore.java

```diff
@@ -464,8 +464,6 @@ public void shutdown() throws NacosException {
         masterTask.cancel(true);
         Loggers.RAFT.warn("stop old raft protocol task for heartbeat task");
         heartbeatTask.cancel(true);
-        Loggers.RAFT.warn("clean old cache datum for old raft");
-        datums.clear();
```

Comment:
This is cached data, not really persistent data，so clear this map it‘s no problem